### PR TITLE
Small changes to get (aerosol) cycling going at C384

### LINKED ISSUE #1738: Task gdasanal failed on C384 gdas cycle run
I'm running the C384 gdas cycle tests. The task `gdasanal` failed with the following error messages:

```
 44: Load the cloud coefficients:
 44: ...Cloud model: CRTM
 44: ...CloudCoeff file: ./crtm_coeffs/CloudCoeff.bin
116: forrtl: severe (174): SIGSEGV, segmentation fault occurred
116: Image              PC                Routine            Line        Source
116: gsi.x              00000000021A80DD  Unknown               Unknown  Unknown
116: libpthread-2.17.s  00002B90BA5B9630  Unknown               Unknown  Unknown
116: libiomp5.so        00002B90B8BB4FD1  Unknown               Unknown  Unknown
116: libiomp5.so        00002B90B8BB4D44  Unknown               Unknown  Unknown
116: libiomp5.so        00002B90B8BB6B55  Unknown               Unknown  Unknown
116: libiomp5.so        00002B90B8BADDE4  Unknown               Unknown  Unknown
116: libiomp5.so        00002B90B8BB02BB  Unknown               Unknown  Unknown
116: gsi.x              00000000021EA19A  Unknown               Unknown  Unknown
116: gsi.x              0000000002190FB8  Unknown               Unknown  Unknown
116: gsi.x              0000000002190EA0  Unknown               Unknown  Unknown
116: gsi.x              0000000002190EA0  Unknown               Unknown  Unknown
116: gsi.x              000000000150283A  crtm_interface_mp         950  crtm_interface.f90
116: gsi.x              00000000012BCA49  rad_setup_mp_setu        2188  setuprad.f90
116: gsi.x              0000000000EDA1CA  gsi_radoper_mp_se         100  gsi_radOper.F90
116: gsi.x              0000000000A937EC  setuprhsall_              490  setuprhsall.f90
116: gsi.x              0000000000EC0D28  glbsoi_                   323  glbsoi.f90
116: gsi.x              0000000000626E39  gsisub_                   200  gsisub.F90
116: gsi.x              00000000004088BD  gsimod_mp_gsimain        2311  gsimod.F90
116: gsi.x              00000000004087FF  MAIN__                    631  gsimain.f90
116: gsi.x              000000000040879E  Unknown               Unknown  Unknown
116: libc-2.17.so       00002B90BB0A1555  __libc_start_main     Unknown  Unknown
116: gsi.x              00000000004086A9  Unknown               Unknown  Unknown
```
I drop the entire log file at the bottom for your review. It seems this failing due to out of memory again. The computing resource were listed below: 

```
<task name="gdasanal" cycledefs="gdas" maxtries="&MAXTRIES;">

        <command>&JOBS_DIR;/anal.sh</command>

        <jobname><cyclestr>&PSLOT;_gdasanal_@H</cyclestr></jobname>
        <account>fv3-cpu</account>
        <queue>batch</queue>
        <partition>hera</partition>
        <walltime>00:50:00</walltime>
        <nodes>40:ppn=4:tpp=10</nodes>
        <native>--export=NONE</native>

        <join><cyclestr>&ROTDIR;/logs/@Y@m@d@H/gdasanal.log</cyclestr></join>
```
[gdasanal.log](https://github.com/NOAA-EMC/global-workflow/files/12053307/gdasanal.log)

### PR DESCRIPTION
**Pull request by CoryMartin-NOAA**
**Description**

Trying both ATM C384/C192 cycling with GSI hybrid-4DEnVar (half res of ops) and ATMA C384 3DVar on Hera, I ran into several problems. This PR solves them (for now!)

Change summary:
1. The aeroanl layouts are now all 8x8, this is just so that we can use the C96 BUMP files that are 8x8 layout for higher resolutions (this is not expected to be a permanent change, and we will have 'optimized' values later)
2. analcalc would fail (Closes #1738 ), this specifies the amount of memory and allows it to complete
3. The first half cycle C384 forecast would fail, changing `nthreads_fv3` to 2 fixes this
4. The epos jobs do not need 14 nodes allocated, this changes it to 2 nodes, and it runs at this resolution in only 3 mins on Hera
5. A bugfix (missing `f` before a f-string) and changed `_res_enkf` to `_res_anl` because `CASE_ANL` is set in `config.aeroanl` but `CASE_ENKF` will not be set by the setup_expt script if you specify `nmem` to be 0  

**Type of change**

- [x] Bug fix (non-breaking change which fixes an issue)

**How Has This Been Tested?**

- [x] Cycled test on Hera ATM
- [x] Cycled test on Hera ATMA

  
**Checklist**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [x] Any dependent changes have been merged and published

### CODE REVIEW

**Review by RussTreadon-NOAA — APPROVED** [review]
Did test the aero changes but look OK to me.

File: parm/config/gfs/config.resources
Comment (WalterKolczynski-NOAA) [inline parm/config/gfs/config.resources]: Should this be changed to 1 for all machines? We can get rid of this special case for Hera and just make it 1 everywhere.

**Review by WalterKolczynski-NOAA — CHANGES_REQUESTED** [review]

File: parm/config/gfs/config.resources
Comment (CoryMartin-NOAA) [reply parm/config/gfs/config.resources]: Probably but I did not test and am not sure why this was 4 and 6 in the first place... It seems like it is not using any threads at all.

**Comment by RussTreadon-NOAA** [comment]
I checked enkfgdas job log files on WCOSS2. The operational enkfgdas_post jobs run with 4 threads.    `getsfcensmeanp.x` is not threaded.   The smoothing sections of `getsigensmeanp_smooth.x` are threaded.  We used to run this code with smoothing on.  We do not exercise the smoothing portion of the code in the current operational enkfgdas_post.   

I don't know why we have 6 threads for Hera.

File: parm/config/gfs/config.resources
Comment (WalterKolczynski-NOAA) [reply parm/config/gfs/config.resources]: Looks like Kate _lowered_ it to 4 after the WCOSS2 port (PR #1070). And before that it had been lowered to 6 from 8 pre-GitHub. Who knows why it was so high to begin with (I would assume memory). Given Hera is the slowest, lowest-mem system, I suspect lowering it everywhere would work.

Comment (WalterKolczynski-NOAA) [reply parm/config/gfs/config.resources]: With Russ's reply, I'm more confident we can just use 1.

Comment (CoryMartin-NOAA) [reply parm/config/gfs/config.resources]: Done.

**Review by WalterKolczynski-NOAA — APPROVED** [review]
Conditionally approved pending CI results. I don't expect any issues with the tests, since these changes mainly impact higher resolutions.

File: parm/config/gfs/config.ufs
Comment (aerorahul) [inline parm/config/gfs/config.ufs:154]: This resolution should be tested more broadly with a S2SW forecast only application.

**Review by aerorahul — COMMENTED** [review]
The changes are ok with me. Though, I think for the change in `config.ufs` for `C384` should be tested in a coupled forecast.

**Comment by CoryMartin-NOAA** [comment]
Don't merge this in yet even if everything passes, the `enkfgdasesfc` job fails running out of time, want to make sure my change I just tried allows it to complete. 6 mins was not enough.

**Review by jiaruidong2017 — APPROVED** [review]
Thanks for including the changes in `config.ufs` and `config.resources` for fixing the failure of the C384 gdas cycle run.

**Review by DavidNew-NOAA — APPROVED** [review]

**Comment by CoryMartin-NOAA** [comment]
I had to extend the walltime for esfc to 8 mins from 6 since it was taking 6-7 mins to complete at this resolution on Hera.

**Comment by WalterKolczynski-NOAA** [comment]
@CoryMartin-NOAA is this ready to go now. then?

**Comment by CoryMartin-NOAA** [comment]
@WalterKolczynski-NOAA yes I think so

**Pull request merged**
